python/pyspark/ml/util.py

```diff
@@ -460,8 +461,8 @@ class DefaultParamsReadable(MLReadable):
     If a :py:class:`Params` class stores all data as :py:class:`Param` values,
     then extending this trait will provide a default implementation of reading saved
     instances of the class. This only handles simple :py:class:`Param` types;
-    e.g., it will not handle :py:class:`Dataset`. See :py:class:`DefaultParamsWritable`,
-    the counterpart to this trait.
+    e.g., it will not handle :py:class:`pyspark.sql.DataFrame`. See
+    :py:class:`DefaultParamsWritable`, the counterpart to this trait.
```

Comment:
I think we can just use `:class:`

Reply:
in this file, :py:class is used everywhere . I don't mind changing the above, but maybe we should then change it everywhere for consistency ?

Reply:
Yeah, let's just leave it. that's fine